tests/js/spec/views/eventsV2/eventView.spec.jsx

```diff
@@ -145,3 +147,30 @@ describe('EventView.getEventsAPIPayload()', function() {
     expect(eventView.getEventsAPIPayload(location).query).toEqual('event.type:csp');
   });
 });
+
+describe('EventView.toNewQuery()', function() {
+  it('outputs the right fields', function() {
+    const eventView = new EventView({
+      id: '2',
+      name: 'best query',
+      fields: [
+        {field: 'count()', title: 'count'},
+        {field: 'project.id', title: 'project'},
+      ],
+      query: 'event.type:error',
+      range: '14d',
+      start: '',
+      end: '',
+      sorts: [{field: 'timestamp', kind: 'desc'}],
+    });
+    const output = eventView.toNewQuery();
+    expect(output.fields).toEqual(['count()', 'project.id']);
+    expect(output.fieldnames).toEqual(['count', 'project']);
+    expect(output.name).toEqual(eventView.name);
+    expect(output.range).toEqual('14d');
```

Comment:
Nit: Would be good to have the `expect()` in the same order as Line 154-164, makes it easier to check.